components/data/librarysize/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { MetaNode } from '@/spec/metanode'
+import python from '@/utils/python'
+import { GeneCountMatrix } from '@/components/data/gene_count_matrix'
+import { norm_icon } from '@/icons'
+import { PlotlyPlot } from '@/components/viz/plotly'
+
+
+
+export const librarysize = MetaNode('librarysize')
```

Comment:
I recommend preserving the current casing of the other metanodes, maybe it should be called `VisualizeLibrarySize`